VocaDbWeb/Views/Event/Details.cshtml

```diff
@@ -216,6 +216,29 @@
 
 @CommentHelpers.LatestCommentsKnockout()
 
+<!--	<div id="tabs" class="js-cloak" data-bind="show, with: comments">

```

Comment:
This will require a bit more code. Can you please remove this change and create another branch?